app/models/liabilitycalculation/Calculation.scala

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.liabilitycalculation
+
+import models.liabilitycalculation.taxcalculation.TaxCalculation
+import play.api.libs.json.{Json, OFormat}
+
+case class Calculation(
```

Comment:
There are some fields that are in swagger and not here. This might be fine with what you need. Some fields I found are: royaltyPayments, notionalTax, pensionContributionReliefs, pensionSavingsTaxCharges, pensionContributionReliefs, pensionSavingsTaxCharges, etc.

Reply:
yep thanks - these are not needed - we've only included the fields that our frontend specifically uses